lib/aws/rails/railtie.rb

```diff
@@ -28,11 +28,18 @@ class Railtie < ::Rails::Railtie
     #
     # @param [Symbol] name The name of the ActionMailer delivery method to
     #   register.
-    # @param [Hash] options The options you wish to pass on to the
-    #   Aws::SES::Client initialization method.
-    def self.add_action_mailer_delivery_method(name = :ses, options = {})
+    # @param [Hash] client_options The options you wish to pass on to the
+    #   Aws::SES[V2]::Client initialization method.
+    def self.add_action_mailer_delivery_method(name = :ses, client_options = {}, mailer = :ses)
```

Comment:
`name` already covers `mailer` here doesn't it? So I think we should re-use that one.

Reply:
Ah sorry.. brain fart. This one is weird. Previously someone can have name :foo for SES mailer V1. We could decide if "name == :sesv2" means also using ses v2 mailer, and tie delivery method name to mailer class. What do you think?

Reply:
Yes I think that 'mailer' parameter is redundant to be honest. I'll amend the code to use "name == :sesv2" as the indicator to use V2.

Reply:
Sounds good, see my comment below about checking for this.